builder/main.py

```diff
@@ -234,8 +242,21 @@ def upload_test_coverage(env):
     # only works for linux for now
     env.shell.exec('curl', '-Os', 'https://uploader.codecov.io/latest/linux/codecov', check=True)
     env.shell.exec('chmod', '+x', 'codecov', check=True)
+    include_args = UniqueList(['source/'])
+    include_args += env.args.coverage_include
+    exclude_args = UniqueList(env.args.coverage_exclude)
+    uploader_args = []
+    for include in include_args:
+        include += '*'
```

Comment:
trivial: this seems to assume that the user passed a path ending with "/" ?

maybe don't add '/' or '*' and simply document the path should be like "source/*"

that also lets people include or exclude specific files, instead of assuming it's always a folder

Reply:
actually ... there may be an issue asking users to pass "*" since it needs to be quoted to avoid the shell doing magic on that character

python's core libraries *should* be good about adding those quotes, but who knows what kind of assumptions our code makes passing around args...

test this branch against aws-c-common (using BUILDER_SOURCE=channels) before doing an actual release with my suggestions

and if it doesn't work, go ahead and keep adding the "*" automatically

Reply:
actually when people want to include/exclude a specific file, the asterisk is also kind needed as the file to upload ends with `.gcov`. (I was checking the `.endwith("/")` to add `*` in the end, and then find out that the files cannot be found).